devctl/config/devctl_json_config.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "Devctl",
+  "org": "HEMSaw",
+  "site": "OK1",
+  "user": "devctl",
+  "db": {
+    "type": "MongoDB",
+    "protocol": "mongodb",
+    "host": "mongodb",
+    "port": 27017,
+    "user": "",
+    "password": ""
+  },
+  "interfaces": [
+    {
+      "enable": "Yes",
+      "device": {
+        "device_uid": "HEMSaw-123456",
```

Comment:
Sed command implemented but commented out. It can be uncommented auto-set the machine UUID. Also, do we need to allow changing the name of this file, or should this just be set as the default?

Reply:
The name of the configuration file must be same for devctl to access. While copying from `./devctl/config/` to `/etc/devctl/config` it  has to be with name `devctl_json_config.json`